### PR TITLE
Add better explanations to tables.ipynb

### PR DESCRIPTION
**Pull request by daniel-sol**
Make tables noetbook more complete, with more data, and examples

### CODE REVIEW

File: examples/tables.ipynb [hunk 1]
```diff
@@ -67,7 +102,73 @@
...
+   "source": [
+    "one_table[\"data\"].keys()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since it is slow to aggregate the tables yourself, and that the tables then would become very large Sumo comes with a service that aggregates the <br>tables, and then splits them up by column. We refer to these objects as aggregated tables even though they are both aggregated and split up.<br> These come in different types, but the most general is the collection, where you get all realizations stacked on top of each other,<br> but you also have access to statistical aggregations such as mean, min,max, std, p10, and p90 <br>\n",
+    "\n",
```
Comment (equinor-ruaj) [inline examples/tables.ipynb:170]: Do we need/want to expose the fact that the aggregated tables are split up to users?

**Review by equinor-ruaj — APPROVED** [review]
lgtm

File: examples/tables.ipynb (hunk 1 above)
Comment (daniel-sol) [reply examples/tables.ipynb]: I think we should, but have modified the text to clarify a bit around this.

**Review by equinor-ruaj — APPROVED** [review]
lgtm

**Pull request merged**